src/BasicMath/MethodAuthenticationHandler.php

```diff
@@ -3,10 +3,13 @@
 
 use SilverStripe\Control\HTTPRequest;
 use SilverStripe\Core\Config\Configurable;
-use SilverStripe\MFA\AuthenticationMethod\AuthenticatorInterface;
-use SilverStripe\MFA\SessionStore;
+use SilverStripe\MFA\Method\Handler\LoginHandlerInterface;
+use SilverStripe\MFA\Store\StoreInterface;
 
-class MethodAuthenticator implements AuthenticatorInterface
+/**
+ * Handles login attempts for the Math Method
+ */
+class MethodAuthenticationHandler implements LoginHandlerInterface
```

Comment:
I think you should call it `LoginHandler` - its namespace is enough to differentiate it from others

Reply:
Oops, missed this one 😅